client/src/i18n/locales/es/index.json

```diff
@@ -221,5 +222,20 @@
   "addAnotherDate": "Otra fecha",
   "success": "¡Registro exitoso!",
   "successText": "Acabas de registrar el historial de asistencia para los niños bajo tu supervisión.",
-  "gotToDashboard": "Ir al panel principal"
+  "gotToDashboard": "Ir al panel principal",
+  "dateError": "Por favor selecciona una fecha",
+
+  "inputAttendance": "Ingresar asistencia",
+  "name": "Nombre",
+  "mon": "mon",
+  "tues": "tues",
+  "wed": "wed",
+  "thurs": "thurs",
+  "fri": "fri",
+  "sat": "sat",
+  "sun": "sun",
+
+  "screenSize": "Tamaño de pantalla no compatible",
+  "incompatibleMsg": "O tu navegador is demasiado pequeño o estás en un aparato movil. Por favor cambia a una computadora o tableta para ver esta pagina.",
+  "noInfo": "no info"
```

Comment:
Should this be in Spanish?

Reply:
"no info" is also the spanish translation. confirmed this with Chelsea